Remkuzo98/Capstone-project-Remy
# deleting files

> #### What does this PR do?
> [what does this PR do?]
> this is the landing page of the website
> #### Description of Task to be completed
> [the tasks that you completed on the feature]
> this page has to be rebuilt in the ordered structure
> #### How should this be manually tested?
>N/A
> #### Any background context you want to provide?
>N/A
> #### What are the relevant Trello story?
> a user should be able to navigate to different pages and see my profile 
> #### Screenshots (if appropriate)
> 
![landing page](https://user-images.githubusercontent.com/53525195/92045862-dfd40680-ed81-11ea-8c3d-04e016b213fc.png)
